source/iterators.tex

```diff
@@ -2408,6 +2411,20 @@
 There are several concepts that group requirements of algorithms that
 take callable objects\iref{func.def} as arguments.
 
+\rSec3[indirectcallable.traits]{Indirect callable traits}
+
+\pnum
+To implement algorithms taking projections,
+it is necessary to determine the projected type of an iterators value type.
+The exposition-only alias template \tcode{indirect-value-t<T>} denotes
```

Comment:
`indirect-value-t<T>` is not an alias template, it's an alias template specialization.
```suggestion
The exposition-only alias template specialization \tcode{indirect-value-t<T>} denotes
```

Reply:
Maybe "For the exposition-only alias template \tcode{\exposid{indirect-value-t}}, \tcode{\exposid{indirect-value-t}<T>} denotes"? We want to say that the template is exposition-only, not that the specialization for `T` is.

Reply:
That's much better. As suggested:
```suggestion
For the exposition-only alias template \exposid{indirect-value-t}, \tcode{\exposid{indirect-value-t}<T>} denotes
```

Reply:
Much better! added as a separate commit